### PR TITLE
don't use require_once in code, use composer autoloading instead

### PR DESCRIPTION
**Pull request by dpfaffenbauer**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -103,6 +103,10 @@
     "elasticsearch/elasticsearch": "2.0.0"
   },
   "autoload": {
+    "files": [
+      "lib/simple_html_dom.php",
```
Comment (brusch) [inline composer.json]: I'm not sure if we should really load this huge library on every request, even if it's not used ... this doesn't feel right to me ...
I think we should instead migrate to the OOP port of simple_html_dom: https://packagist.org/packages/sunra/php-simple-html-dom-parser

Comment (dpfaffenbauer) [reply composer.json]: To be honest: it doesn’t hurt PHP at all loading it. It’s getting opcached anyway. 

But: if the OOP port works, I am happy with that as well

Comment (brusch) [reply composer.json]: So are you updating your PR?

Comment (dpfaffenbauer) [reply composer.json]: yep, later today

Comment (brusch) [reply composer.json]: Seems that `sunra/php-simple-html-dom-parser` has still some issues with PHP 7.3 (there are some open PRs for that), so probably not an alternative for now.

**Comment by dpfaffenbauer** [comment]
ok, happy with that

**Comment by brusch** [comment]
Changed Simple DOM HTML as well. 
It's used in 90% of the requests anyway, so no real benefit ...
Thanks again!

**Pull request closed without merging**